test-clients/src/main/java/com/hedera/services/bdd/suites/schedule/ScheduleSignSpecs.java

```diff
@@ -71,17 +75,119 @@ public static void main(String... args) {
 						scheduleSigIrrelevantToSchedulingTxn(),
 						overlappingKeysTreatedAsExpected(),
 						retestsActivationOnSignWithEmptySigMap(),
-						triggersUponFinishingPayerSig(),
+						basicSignatureCollectionWorks(),
+						addingSignaturesToExecutedTxFails(),
+						addingSignaturesToNonExistingTxFails(),
+						addingSignatureByNonRequiredSignerFails(),
+						addingSignatureByNonRequiredSignerFails2(),
+						triggersUponFinishingPayerSig()
 				}
 		);
 	}
 
+	private HapiApiSpec basicSignatureCollectionWorks() {
+		var txnBody = cryptoTransfer(tinyBarsFromTo("sender", "receiver", 1));
+
+		return defaultHapiSpec("BasicSignatureCollectionWorks")
+				.given(
+						updateScheduleExpiryTimeSecs,
+						overriding("scheduling.whitelist", "CryptoTransfer"),
+						cryptoCreate("sender"),
+						cryptoCreate("receiver").receiverSigRequired(true),
+						scheduleCreate("basicXfer", txnBody)
+				)
+				.when(
+						scheduleSign("basicXfer").withSignatories("receiver")
+				)
+				.then(
+						getScheduleInfo("basicXfer")
+								.hasSignatories("receiver")
+				);
+	}
+
+	private HapiApiSpec addingSignatureByNonRequiredSignerFails() {
+		var txnBody = cryptoTransfer(tinyBarsFromTo("sender", "receiver", 1)).signedBy("somebody");
```

Comment:
you do not need the `signedBy` method. It only adds additional complexity but is not required for the test. Someoen reading the test might be confused because of the unnecessary additions